### PR TITLE
Remove mentions of cost estimation pre-alpha

### PR DESCRIPTION
**Pull request by davidcelis**

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -53,8 +53,6 @@ const (
 	RunApplying           RunStatus = "applying"
 	RunCanceled           RunStatus = "canceled"
 	RunConfirmed          RunStatus = "confirmed"
-	RunCostEstimated      RunStatus = "cost_estimated"
-	RunCostEstimating     RunStatus = "cost_estimating"
```
Comment (svanharmelen) [inline run.go]: I think we should keep these states as in some cases they can be returned right?

File: helper_test.go [hunk 2]
```diff
@@ -343,7 +343,7 @@ func createPlannedRun(t *testing.T, client *Client, w *Workspace) (*Run, func())
 		}
 
 		switch r.Status {
-		case RunPlanned, RunCostEstimated, RunPolicyChecked, RunPolicyOverride:
```
Comment (svanharmelen) [inline helper_test.go]: I think we should keep this state as in some cases it can be returned right?

File: run.go (hunk 1 above)
Comment (thrashr888) [reply run.go]: I think that, for now, these are not possible to see in production. If they end up changing, we'll have to remove/replace them anyway, which might be easier to PR as a group.

File: helper_test.go (hunk 2 above)
Comment (thrashr888) [reply helper_test.go]: See other comment.

File: run.go (hunk 1 above)
Comment (svanharmelen) [reply run.go]: Check 👍🏻

**Pull request merged**